shell/platform/embedder/tests/embedder_config_builder.cc

```diff
@@ -349,52 +299,4 @@ UniqueEngine EmbedderConfigBuilder::SetupEngine(bool run) const {
   return UniqueEngine{engine};
 }
 
-#ifndef SHELL_ENABLE_GL
```

Comment:
Everything below here are just methods on the appropriate `EmbedderTestContext` subclass. People who need GL-specific methods can get them from the GL subclass, etc.